en/docs/get-started/quick-start-guide.md

```diff
@@ -136,262 +146,192 @@ Let's test the `readingList` REST API via Choreo's Open API Console by following
       }
     ```
    
-5. Click **Execute**.
+6. Click **Execute**.
 
     Check the **Server Response** section. On successful invocation, you will receive the `200` HTTP code.
 
 Similarly, you can expand and try out the **GET** and **DELETE** methods.
 
 
-### Step 1.4: Publish the REST API
+### Step 1.4: Publish the service
 
-Now that your API is tested, let's publish it and make it available for applications to consume.
+Now you can publish the tested REST endpoint to make it available for the web application to consume.
 
 #### Step 1.4.1: Update the CORS configuration
 
-The application developer in this scenario calls the API from a different domain to Choreo (i.e., Vercel). By default, web browsers block these calls for security reasons. To enable the application to call the API, update the CORS configuration as follows:
+Since you expect to deploy the web application in Vercel, you would want to call the **Readinglist** backend service deployed in Choreo through the web application on a Vercel domain. By default, web browsers block such calls for security reasons. To enable the web application to call the backend service, you must update the CORS configuration as follows:
```

Comment:
"Since you expect to deploy the web application in Vercel" -> We don't set this expectation before hand do we? Shall we rephrase this to imply that you will be doing this, rather than making it sound as it's an expectation of the user?

Reply:
+1
Rephrase the sentence.